crates/madara/node/src/cli/mod.rs

```diff
@@ -245,6 +249,14 @@ impl RunCmd {
         self.name.as_ref().expect("Name was just set")
     }
 
+    pub fn check_mode(&self) -> anyhow::Result<()> {
+        if (self.sequencer | self.full) ^ (self.full | self.devnet) ^ (self.sequencer | self.devnet) {
```

Comment:
Why do this through binary operations? Doesn't this also mean that if `self.full == true` and no other this will return an error?
```suggestion
        if !self.sequencer && !self.full && !self.devnet {
```

Reply:
It works for all cases, I just did it this way because I thought about the logic output and not through short circuiting, but you solution is waaaaaay cleaner